test/isotest.js

```diff
@@ -21,23 +21,19 @@ const isReactClass = (Component) => {
     ;
 };
 
-const customProps = {
-    Hotkey: { global: true },
-    KeyCombo: { combo: "?" },
-};
-
-const customChildren = {
-    Popover: [React.createElement("div")],
-};
-
-const shallowRenderer = ReactTestUtils.createRenderer();
+const shallowRenderer = createRenderer();
 
-Object.keys(Components).forEach((ComponentKey) => {
-    describe(ComponentKey, () => {
+/**
+ * Tests that each ComponentClass in Components can be isomorphically rendered on the server.
+ * @param Components        {{ [componentName: string]: any }}              main export from package
+ * @param customProps       {{ [componentName: string]: any}}               custom props per component
+ * @param customChildren    {{ [componentName: string]: React.ReactNode }}  custom children per component
+ */
+module.exports = function isotest(Components, customProps, customChildren) {
+    Object.keys(Components).forEach((ComponentKey) => {
         const Component = Components[ComponentKey];
-
         if (isReactClass(Component)) {
-            it("can be server-rendered", () => {
+            it(`<${ComponentKey}> can be server-rendered`, () => {
```

Comment:
I haven't had a look at all of the changes yet, but in my experience, console outputs can become interlaced when running in parallel/asynchronously. I'd imagined this to be the reason why the tests were set up to run in series when I first found then.